dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/enrollment/EnrollmentRequestParamsMapperTest.java

```diff
@@ -198,4 +204,66 @@ void testMappingOrderParamsNoOrder() throws BadRequestException {
 
     assertIsEmpty(params.getOrder());
   }
+
+  @Test
+  void shouldFailWhenOrgUnitSuppliedAndOrgUnitModeAccessible() {
+    RequestParams requestParams = new RequestParams();
+    requestParams.setOrgUnits(Set.of(UID.of(ORG_UNIT_1_UID)));
+    requestParams.setOrgUnitMode(ACCESSIBLE);
+
+    Exception exception = assertThrows(BadRequestException.class, () -> mapper.map(requestParams));
+
+    assertEquals(
```

Comment:
You could use `assertStartsWith` or `assertContains` to assert on parts of the message to be sure it is actually the case you wanted to test. This way you might still have a bit more flexibility in changing the message without always having to change the test.

If we move the function to RequestParamUtils we can also test it there or at least most of the cases so we don't have to repeat all the tests.